src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -22,32 +23,30 @@ def new_patchinfo
       flash[:error] = "Patchinfo not found for #{params[:project]}"
       redirect_to(controller: 'package', action: 'show', project: @project, package: @package) && return
     end
-
-    read_patchinfo
-    @binaries.each { |bin| @binarylist.delete(bin) }
+    redirect_to edit_patchinfo_path(project: @project, package: @package)
   end
 
-  def updatepatchinfo
+  def fetch_issues
```

Comment:
Are we really fetching issues here? It seems more like it's updating.

Reply:
We'll re-think this, maybe 'fetch' is not the best word.

Reply:
Changed to `update_issues`